shuup_tests/api/test_basket_api.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
```

Comment:
We also have to test here request using, probably the `requests` module. APIClient is stateful, it will retain cookies and thus session. Make requests without authentication to prove it is possible to create a basket and add/remove items without authentication.